lib/actions/user.js

```diff
@@ -105,3 +162,30 @@ export function createOrUpdateUser (userData) {
     }
   }
 }
+
+/**
+ * Deletes a logged-in user's monitored trip,
+ * then, if that was successful, refreshes the redux monitoredTrips state.
+ */
+export function deleteUserMonitoredTrip (id) {
+  return async function (dispatch, getState) {
+    const { otp, user } = getState()
+    const { otp_middleware: otpMiddleware = null } = otp.config.persistence
+
+    if (otpMiddleware) {
+      const { accessToken, loggedInUserMonitoredTrips: trips } = user
+      const deleteResult = await deleteTrip(otpMiddleware, accessToken, id)
+
+      if (deleteResult.status === 'success') {
```

Comment:
Let's just dispatch `fetchTrips`.

Reply:
Address in https://github.com/opentripplanner/otp-react-redux/pull/174/commits/b0395b10a66ab3639f243d25250007599aaf70dd.